ui/components/multichain/pages/connections/__snapshots__/connections.test.tsx.snap

```diff
@@ -293,17 +293,17 @@ exports[`Connections Content should render correctly 1`] = `
                     <div
                       class="mm-box currency-display-component mm-box--display-flex mm-box--flex-wrap-wrap mm-box--align-items-center"
                       data-testid="first-currency-display"
-                      title="$880.18 USD"
```

Comment:
I did not track why the fiat balance was actually this low in the original snapshot.

However, based on [this account's balance](https://github.com/MetaMask/metamask-extension/blob/develop/test/data/mock-state.json#L658), the actual balance should be:
```text
0x346ba7725f412cbfdb # Hex (WEI)

# Convert to decimal
966987986469506500000 # Decimal (WEI)

# Convert to ETH
966.9879864695065 # Decimal (ETH)
```

So here again, I guess the snapshot was wrong too. Also, the same [comment](https://github.com/MetaMask/metamask-extension/pull/26224/files#r1698995699) applies here, so no more fiat.